src/main/java/org/sagebionetworks/web/client/widget/team/JoinTeamWidget.java

```diff
@@ -164,9 +212,8 @@ public void onFailure(Throwable caught) {
 	}
 	
 	@Override
-	public void sendJoinRequest(String message, boolean isAcceptingInvite) {
+	public void sendJoinRequest(String message) {
 		this.message = message;
-		this.isAcceptingInvite = isAcceptingInvite;
```

Comment:
This is fine on the Team page, but what if the Join button is in a wiki?
Does the user receive any feedback after successfully being added to the team?